src/RoadRegistry.BackOffice.EventHost/Program.cs

```diff
@@ -62,12 +61,11 @@ public static async Task Main(string[] args)
 
                     var loggerConfiguration = new LoggerConfiguration()
                         .ReadFrom.Configuration(hostContext.Configuration)
-                        .WriteTo.Console()
+                        .WriteTo.Console(new RenderedCompactJsonFormatter())
```

Comment:
Shouldnt we read this from config instead?

Reply:
done